cfgov/ask_cfpb/scripts/export_ask_data.py

```diff
@@ -0,0 +1,143 @@
+from __future__ import unicode_literals
+
+import datetime
+
+import HTMLParser
+from django.utils import html
+
+from paying_for_college.csvkit import csvkit
+
+from ask_cfpb.models import Answer
+
+html_parser = HTMLParser.HTMLParser()
+
+HEADINGS = [
+    'ASK_ID',
+    'Question',
+    'ShortAnswer',
+    'Answer',
+    'URL',
+    'SpanishURL',
+    'Topic',
+    'SubCategories',
+    'Audiences',
+    'RelatedQuestions',
+    'RelatedResources',
+]
+
+SPANISH_HEADINGS = [
+    'ASK_ID'.encode('UTF16'),
+    'SpanishURL'.encode('UTF16'),
+    'SpanishAnswer'.encode('UTF16'),
+]
+
+
+def clean_and_strip(data):
+    unescaped = html_parser.unescape(data)
+    return html.strip_tags(unescaped).strip()
+
+
+def clean_and_strip_spanish(data):
+    unescaped = html_parser.unescape(data)
+    return html.strip_tags(
+        unescaped).strip().replace(
+        '\t', ' ').replace(
+        '\n', ' ').replace(
+        '\r', ' ')
```

Comment:
Is there a possibility of these characters needing to be replaced for non-spanish? Or: can the same `clean_and_strip` function be used for both languages? It only seems to be used when `assemble_output(spanish_only=True)`.